activerecord/lib/active_record/persistence.rb

```diff
@@ -460,14 +460,19 @@ def reload(options = nil)
     #   ball = Ball.new
     #   ball.touch(:updated_at)   # => raises ActiveRecordError
     #
-    def touch(*names)
+    def touch(*args)
       raise ActiveRecordError, "cannot touch on a new record object" unless persisted?
 
       attributes = timestamp_attributes_for_update_in_model
+      time_opt, names = args.partition{|arg| arg.kind_of?(Hash) && arg.has_key?(:time)}
```

Comment:
Ruby style here should have spaces after the curly braces

`time_opt, names = args.partition{ |arg| arg.kind_of?(Hash) && arg.has_key?(:time) }`